src/sentry/workflow_engine/models/action.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, region_silo_model, sane_repr
+
+
+@region_silo_model
+class Action(DefaultFieldsModel):
+    """
+    Actions are actions that can be taken if the conditions of a DataConditionGrou are satisfied.
+    Examples include: detectors emitting events, sending notifications, creating an issue in the Issue Platform, etc.
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+    __repr__ = sane_repr("workflow_id", "type")
+
+    # TODO (@saponifi3d): Don't hardcode these values
+    class Type(models.TextChoices):
+        Notification = "SendNotificationAction"
+        TriggerWorkflow = "TriggerWorkflowAction"
```

Comment:
Will these be full paths to the actions in practice, similar to issue rules?

Reply:
That part is a bit unclear to me, i think it'd be cool if we could derive the information from an `init` file or something and automate it as much as possible. if we did it that way w/o the full path, i think we'd need a registry underneath it to link this type to the class itself.

If we did the full id, could likely omit that registry? (i believe we could just get the module by the module id, just want to confirm)

Reply:
Yeah, either way can work out ok. I guess one thing that's nice about a registry is that it's less susceptible to accidental breakage when people move things around.